### PR TITLE
Release spanner 1.4.0

### PR DESCRIPTION
**Pull request by crwilcox**
This pull request was generated using releasetool.

Fixes #5466 5465

### CODE REVIEW

File: spanner/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,21 @@
 
 [1]: https://pypi.org/project/google-cloud-spanner/#history
 
+## 1.4.0
+
+### Implementation Changes
+- Ensure that initial resume token is bytes, not text. (#5450)
+- Prevent process_read_batch from mutating params (#5416)
+- Avoid overwriting '__module__' of messages from shared modules. (#5364)
+- BigTable: improve read rows validation performance (#5390)
```
Comment (tseaver) [inline spanner/CHANGELOG.md]: Stray, whitespace only:  drop this one.

File: spanner/CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,21 @@
...
+
+### New Features
+- Modification to enable 3.7 #5282 (#5288)
```
Comment (tseaver) [inline spanner/CHANGELOG.md]: Imperative mode; also, just link PR:  `- Add support for Python 3.7 (#5288)`

File: spanner/CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,21 @@
 
 [1]: https://pypi.org/project/google-cloud-spanner/#history
 
+## 1.4.0
+
+### Implementation Changes
```
Comment (tseaver) [inline spanner/CHANGELOG.md]: Shouldn't this be `### Bug fixes`?

Comment (tseaver) [reply spanner/CHANGELOG.md]: Or `### Bugs fixed`?

Comment (theacodes) [reply spanner/CHANGELOG.md]: Nope, our template is "Implementation Changes".

**Comment by crwilcox** [comment]
Release is at https://github.com/GoogleCloudPlatform/google-cloud-python/releases/tag/spanner-1.4.0

**Pull request merged**